index.js

```diff
@@ -142,7 +238,23 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+//Return movies.title of the movie with the highest movies.boxOffice amount.
+//Convert movies.boxOffice into a number to compare to others.
+
+  let biggestMovie = 0;
+
+
+  for(let i = 0; i < movies.length; i++) {
+    let boxOfficeNumber = parseFloat(movies[i].boxOffice.replace (/\$|,/g, ''));
+    if(Number(boxOfficeNumber) > biggestMovie) {
+      biggestMovie = Number(boxOfficeNumber)
+      console.log(biggestMovie)
+      return movies[i].title
```

Comment:
Since this return is happening inside our if block, the first time our if statement is true we're going to exit our function. I don't think that's what we want to be doing.